pkg/client/rootd/session.go

```diff
@@ -591,6 +591,11 @@ func (s *Session) onClusterInfo(ctx context.Context, mgrInfo *manager.ClusterInf
 		}
 
 		s.clusterSubnets = subnet.Unique(subnets)
+	}
+	// NOTE: When we can access traffic manager by pod ip, we set proxyCluster to false.
+	// Howevery, if you need and can't access cluster by service,
+	// you can use alsProxySubnets to enable proxy service subnets.
+	if s.proxyCluster || len(s.alsoProxySubnets) > 0 {
```

Comment:
I consider this a quick-fix. The real fix is to discover what subnets that are accessible and what subnets that aren't, and then only map the latter. The `alsoProxySubnets` shouldn't be needed in order to accomplish that.

Reply:
Agree.

We should test pod subnet and service subnet for accessibility separately.
However, i am not sure if we should save the manager service ip to test accessible of service subnet.

The bug will completely block the user's use of telepresence, a quick fix might be necessary.
At the same time, from the standpoint of my own habits:
if i set `alsoProxySubnet`, it should take affect, regardless of whether the cluster is accessible or not.